.github/workflows/clear-cache.yml

```diff
@@ -0,0 +1,31 @@
+name: Clear branch cache
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch_number:
+        description: "Branch number"
```

Comment:
```suggestion
        description: "PR number"
```

The cache is associated with the PR number. So calling it PR number makes it clearer, imo

Reply:
agree, although it's kinda WIP :)

Reply:
Yes, yes, it was not a formal review, just a suggestion